crypto/encode_decode/decoder_lib.c

```diff
@@ -225,7 +276,7 @@ int OSSL_DECODER_CTX_add_extra(OSSL_DECODER_CTX *ctx,
      * what the existing ones want as input, and push those newly fetched
      * decoders on top of the same stack.
      * Then it does the same again, but looping over the newly fetched
-     * decoders, until there are no more encoders to be fetched, or
+     * decoders, until there are no more decoders to be fetched, or
      * when we have done this 10 times.
```

Comment:
Maybe get rid of the magic number of 10 and replace with a define. (and in this comment)